docs/modules/objectives/other.mdx

```diff
@@ -5,18 +5,16 @@ title: Other Gamemodes
 
 ### Ghost Squadron
 
-Enables the ghostsquadron gamemode and all its features.
-The default ghostsquadron classes can be found [here](https://raw.githubusercontent.com/OvercastNetwork/maps.oc.tc/master/Blitz/GS/gs.xml).
-Ghost squadron maps set a custom gamemode title in the `gs.xml` with the `<map game="Ghost Squadron">` attribute.
-When not using the default GS XML include a custom title should be set.
-
-`NOTE:` This module must be used in conjunction with another gamemode.
+The ghostsquadron gamemode can be quicky adapted in conjunction with the Blitz gamemode.
+The default ghostsquadron classes and setup can be found [here](https://raw.githubusercontent.com/OvercastNetwork/maps.oc.tc/master/Blitz/GS/gs.xml).
+Ghost squadron maps are set with a custom gamemode title using the `<game>` tag.
```

Comment:
The code that made ghost squadron what it was doesn't exist anymore (e.g. Demo's landmines, Tracker's compass, and Spider's spider-sense). This is what `<ghostquadron/>` did in the XML -- all the special class abilities that were unique to GS. This should just be removed altogether, in my opinion.

Reply:
I do remember the spider sense thing you were talking about, though it looks that tracker and spider were probably additional classes added on OCN that weren't included in the original map. We could probably cut down the unworking classes and make a nice working example.